Stronger/Resources/Localizable.xcstrings

```diff
@@ -218,8 +234,39 @@
         }
       }
     },
+    "Goal Resistance" : {
+
+    },
+    "Goal Weight" : {
+
+    },
+    "GOAL_RESISTANCE" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Goal Resistance"
+          }
+        }
+      }
+    },
+    "GOAL_WEIGHT" : {
+      "extractionState" : "manual",
```

Comment:
Please be sure that you always use an automatic extraction setup, you can change this in the Xcode localization file view in the inspector on the right side.